AGENTS.md

```
@@ -778,761 +262,56 @@ Task (Team + Workspace) → Subtasks (messages/steps)
-**Tech:** Python, Docker SDK, FastAPI, APScheduler
-
-**Key components:**
-- `scheduler/scheduler.py` - Periodic task fetching (online/offline)
-- `executors/docker/executor.py` - Docker container lifecycle
-- `clients/task_api_client.py` - Backend API communication
-
-**Environment variables:**
-- `TASK_API_DOMAIN`, `EXECUTOR_IMAGE`, `NETWORK`
-- `MAX_CONCURRENT_TASKS` (default: 30)
-- `PORT_RANGE_MIN/MAX` (10000-10100)
+| Agent | Key Features |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)